packages/apify/src/actor.ts

```diff
@@ -519,20 +520,20 @@ export async function callTask(taskId: string, input?: Dictionary | string, opti
         ...callTaskOpts
     } = options;
 
-    // @ts-ignore should this be public?
+    // TODO(vladfrangu): should we move users to this in v3?
```

Comment:
The options in client should match the API params exactly. That's a requirement.

The functions in the SDK should ideally just expose the client functions and work exactly the same. And since the client needs to match the API...

Reply:
Ok, then let's align this with the client in v3 - this PR is already too huge, so let's keep this for another one.